src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/AddCommentStep.java

```diff
@@ -25,15 +27,25 @@
   @Getter
   private final String idOrKey;
 
+  @Deprecated
   @Getter
   private final String comment;
 
+  @Getter
+  private Object input;
+  
+  @Deprecated
   @DataBoundConstructor
   public AddCommentStep(final String idOrKey, final String comment) {
     this.idOrKey = idOrKey;
     this.comment = comment;
   }
 
+  @DataBoundSetter
```

Comment:
Add `@DataBoundSetter` to the input field, that should be fine, we don't need this setter. Same this applies for Edit step too. 

```java
@Getter 
@DataBoundSetter
private Object input
```